pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotDriver.java

```diff
@@ -37,16 +38,18 @@
 public class PinotDriver implements Driver {
   private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PinotDriver.class);
   private final String SCHEME = "pinot";
+  public static final String TENANT = "tenant";
 
   @Override
   public Connection connect(String url, Properties info)
       throws SQLException {
     try {
       LOGGER.info("Initiating connection to database for url: " + url);
       PinotClientTransport pinotClientTransport = new JsonAsyncHttpPinotClientTransportFactory().buildTransport();
-      List<String> brokerList = DriverUtils.getBrokersFromURL(url);
       String controllerUrl = DriverUtils.getControllerFromURL(url);
-      return new PinotConnection(brokerList, controllerUrl, pinotClientTransport);
+      Preconditions.checkArgument(info.containsKey(TENANT), "Pinot tenant missing in the properties");
```

Comment:
not needed, you can use the default tenant name and go with it.

Reply:
done